pyfive/dataobjects.py

```diff
@@ -648,6 +657,8 @@ def is_dataset(self):
         """ True when DataObjects points to a dataset, False for a group. """
         return len(self.find_msg_type(DATASPACE_MSG_TYPE)) > 0
 
+UNLIMITED_SIZE = struct.unpack('<Q', b'\xff\xff\xff\xff\xff\xff\xff\xff')[0]
```

Comment:
easier written `(1 << 64)  - 1`

Reply:
Fair enough :) this was copy-paste from some earlier code, which has since been refactored to be 
```python
UNDEFINED_ADDRESS = 0xffffffffffffffff
```
(https://github.com/jjhelmus/pyfive/blob/master/pyfive/core.py#L10), which could also be re-used here as a constant.